### ISSUE TITLE
Shell script should offer specification of output filename

### ISSUE THREAD

**Issue by cortner** [ACEsuit/ACEpotentials.jl#242]
Currently the shell script works as follows
```shell
julia --project=. runfit.jl -p model_spec.json
```
with the output file specified in `model_spec.json`.  There should be another argument that overrides the output file spec, e.g. 
```shell
julia --project=. runfit.jl -p model_spec.json -o result.json
```

**Comment by cortner** [comment]
@iamveronika  -- could you make a PR that adresses this?

**Comment by cortner** [comment]
maybe coordinate with @CheukHinHoJerry  - to see how he envisions using the runfit script.

**Comment by CheukHinHoJerry** [comment]
`julia --project=. runfit.jl -p model_spec.json -o result_folder` seems to be the best for me, and then maybe primarily in the `output` section of `model_spec.json` we can support a few easy tests
```
  "output": {
    "model": "opted_model",
    "dimer": false,
    "error_table": true,
    "scatter_error": false
  }
```
and we output everything into `result_folder` together with the data needed to reproduce those plots in a `result_folder/result.json` file.

**Comment by cortner** [comment]
and maybe the default model filename can just be `model.json`, unless otherwise specified in the model specification dictionary?

**Comment by cortner** [comment]
I think something like this will be great.

**Comment by cortner** [comment]
and can we take a default output folder to be `result_model_spec` where `model_spec` is the input filename? (of `-o` is not specified)

**Comment by CheukHinHoJerry** [comment]
yes same thought, i will create a PR after finishing this items.

**Comment by cortner** [comment]
great, please make sure to merge latest `main` into your branch.

**Comment by cortner** [comment]
Now addressed by #254
